### PR TITLE
Merge 1.7.3.x into develop

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Merged 1.7.3.x into develop
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | n/a
| How to test?  | No testing required

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8623)

### CODE REVIEW

**Review by Quetzacoalt91 — APPROVED** [review]

File: app/config/config.yml [hunk 1]
```diff
@@ -60,6 +60,8 @@ twig:
         '%admin_page%/Configure/AdvancedParameters': AdvancedParameters
     globals:
         webpack_server:   false
+    paths:
+        '%admin_page%/Configure/AdvancedParameters': AdvancedParameters
```
Comment (Quetzacoalt91) [inline app/config/config.yml]: To be removed, we have a duplicate here.

**Review by Quetzacoalt91 — DISMISSED** [review]

File: app/config/config.yml (hunk 1 above)
Comment (eternoendless) [reply app/config/config.yml]: Good catch!

**Review by mickaelandrieu — APPROVED** [review]

**Review by mickaelandrieu — DISMISSED** [review]
e2e tests are broken

**Comment by Quetzacoalt91** [comment]
@fatmaBouchekoua, it seems we have some issue to close the welcome onboarding popup. Have you already heard of it?

**Comment by fatmaBouchekoua** [comment]
@Quetzacoalt91, the problem is not related to the module Welcome because it was removed from the modules folder before installing.

**Comment by Quetzacoalt91** [comment]
I wouldn't be so sure:
```
  1) Allscenario The Product Creation Create new product should click on the <add new product> button:

     Uncaught unknown error: Element <li class="maintab  has_submenu" id="subtab-AdminCatalog" data-submenu="9">...</li> is not clickable at point (105, 165). Other element would receive the click: <div class="onboarding-popup bootstrap">...</div>

  Error: An unknown server-side error occurred while processing the command.

      at elementIdClick("0.9599665873899925-5") - click.js:20:22
```

I guess the module is downloaded again during installation. Anyway, the script was supposed to close the modal if found, did we remove that part?

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Merging even if tests are failing, will be resolved in a specific PR

**Pull request merged**